fearless/Common/Helpers/DerivationPathConstants.swift

```diff
@@ -7,4 +7,5 @@ struct DerivationPathConstants {
     static let hardPasswordPlaceholder = "//hard///password"
     static let defaultEthereum = "//44//60//0/0/0"
     static let zerosEthereum = "//00//00//0/0/0"
+    static let testEthereum = "//12//34//5/6/7"
```

Comment:
I would test empty derivation path also, and please add TODO for when we add no fixed length deviation path support